exercises/ansible_rhel/1.5-handlers/README.md

```diff
@@ -4,304 +4,246 @@
-
-Run it and examine the output. The expected outcome: The task is skipped on node1, node3 and the ansible host (your control host) because they are not in the ftpserver group in your inventory file.
+    - name: Create a new user
+      ansible.builtin.user:
+        name: "{{ user_name }}"
+        state: present
+        create_home: true
 
-```bash
-TASK [Install FTP server when host in ftpserver group] *******************************************
-skipping: [ansible-1]
-skipping: [node1]
-skipping: [node3]
-changed: [node2]
+    - name: Install Apache on web servers
+      ansible.builtin.dnf:
+        name: "{{ package_name }}"
+        state: present
+      when: inventory_hostname in groups['web']
 ```
 
-### Step 2 - Handlers
-
-Sometimes when a task does make a change to the system, an additional task or tasks may need to be run. For example, a change to a service’s configuration file may then require that the service be restarted so that the changed configuration takes effect.
-
-Here Ansible’s handlers come into play. Handlers can be seen as inactive tasks that only get triggered when explicitly invoked using the "notify" statement. Read more about them in the [Ansible Handlers](http://docs.ansible.com/ansible/latest/playbooks_intro.html#handlers-running-operations-on-change) documentation.
-
-As a an example, let’s write a playbook that:
+In this example, `inventory_hostname in groups['web']` is the conditional statement. `inventory_hostname` refers to the name of the current host that Ansible is working on in the playbook. The condition checks if this host is part of the `web` group defined in your inventory file. If true, the task will execute and install Apache on that host.
 
-* manages Apache’s configuration file `/etc/httpd/conf/httpd.conf` on all hosts in the `web` group
-* restarts Apache when the file has changed
+### Step 3 - Handlers
 
-First we need the file Ansible will deploy, let’s just take the one from node1. Remember to replace the IP address shown in the listing below with the IP address from your individual `node1`.
+Handlers are used for tasks that should only run when notified by another task. Typically, they are used to restart services after a configuration change.
 
-```bash
-[student@ansible-1 ansible-files]$ scp node1:/etc/httpd/conf/httpd.conf ~/ansible-files/files/.
-httpd.conf
-```
+Let's say we want to ensure the firewall is configured correctly on all web servers and then reload the firewall service to apply any new settings. We'll define a handler that reloads the firewall service and is notified by a task that ensures the desired firewall rules are in place:
 
-Next, create the Playbook `httpd_conf.yml`. Make sure that you are in the directory `~/ansible-files`.
 
 ```yaml
 ---
-- name: Manage httpd.conf
-  hosts: web
+- name: Basic System Setup
+  hosts: all
   become: true
-  tasks:
+  .
```

Comment:
Extra dots (".") in 3 lines in the yaml between "tasks" and the install firewalld.

Reply:
this is to show its a snippet not the full file.